spec/support/api/schemas/artists.json

```diff
@@ -25,15 +25,27 @@
           "spotify_uri": {
             "id": "http://spotify-apprentice-app.com/artists/0/spotify_uri",
             "type": "string"
+          },
+          "song_ids": {
+            "id": "http://spotify-apprentice-app.com/artists/0/song_ids",
+            "type": "array",
+            "items": {
```

Comment:
yup - usually that info goes at the top before items, to keep all that meta-info organized nicely. no need to change, just FYI